docs/pages/style-spec.js

```diff
@@ -966,7 +966,11 @@ export default class extends React.Component {
                             <a id='transition' className='anchor'></a>
                             <h2><a href='#transition' title='link to transition'>Transition</a></h2>
                             <p>
-                                A style's <code>transition</code> property provides global transition defaults for that style.
+                                A <code>transition</code> property controls timing for the interpolation between a transitionable style
```

Comment:
It might be nice to include same styling for `transitionable` that we do in the rest of the docs here 🤔
```html
<em className='quiet'><span className='icon opacity inline quiet' />transitionable </em>
```

Reply:
Hm, I think 
_transitionable_
denotes that a property can visually/functionally transition between values, whereas
`transition`
is a literal property key that dictates how to perform a transition, so I'm inclined to keep this as `<code>transition</code>`